src/applications/disability-calculator/components/CalculatedDisabilityRating.jsx

```diff
@@ -43,7 +42,9 @@ class CalculatedDisabilityRating extends React.Component {
           then use this VA disability rating to determine your monthly
           disability compensation payment.
         </p>
-        <a href="#">Find your monthly payment amount</a>
+        <a href="https://www.benefits.va.gov/compensation/rates-index.asp">
+          Find your monthly payment amount
+        </a>
```

Comment:
I went ahead and plugged in an href here, but it's determined by our launch date and whether certain content pages are published by then or not. I wrote a ticket to track updating it, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19138.

Reply:
Should this open in a new tab?

Reply:
I looked around the site and I saw links to eBenefits sometimes open in new tabs, sometimes not. I'm going to change this though because based on the context I think I would expect it to open in a new tab 👍